user-guide/modules/ROOT/pages/boost-history.adoc

```diff
@@ -1,29 +1,29 @@
 = History of the Boost C++ Library
 :navtitle: History of Boost
 
-This topic aims to provide an overview of the origins, growth, and impact of the Boost pass:[C++] Library on the programming community.
+This topic aims to provide an overview of the origins, growth, and impact of the Boost Library on the programming community.
 
 == Introduction
 
-The Boost pass:[C++] Library is a highly influential and widely used collection of libraries, designed to extend and complement the functionality provided by the pass:[C++] Standard Library. Since its original https://www.boost.org/users/proposal.pdf[Proposal] in 1998, the Boost pass:[C++] Library has played a significant role in shaping the evolution of the pass:[C++] language and its ecosystem.
+The Boost Library is a highly influential and widely used collection of libraries, designed to extend and complement the functionality provided by the pass:[C++] Standard Library. Since its original https://www.boost.org/users/proposal.pdf[Proposal] in 1998, the Boost Library has played a significant role in shaping the evolution of the pass:[C++] language and its ecosystem.
```

Comment:
> The Boost Library

It's either "Boost" or "The Boost Libraries"

Reply:
@alandefreitas As far as I can see this change is in my last commit - not sure which one you want me to remove?

Reply:
Your latest commit says:

> The Boost Library is a highly influential and widely used collection of libraries, designed to extend and complement the functionality provided by the pass:[C++] Standard Library. Since its original https://www.boost.org/users/proposal.pdf[Proposal] in 1998, the Boost Library has played a significant role in shaping the evolution of the pass:[C++] language and its ecosystem.

![Uploading image.png…]()

Reply:
It was "The Boost pass:[C++] Library" and now it is "The Boost Library"

Reply:
@alandefreitas  are you saying I should change this to "The Boost Libraries..." ?

Reply:
Yes. Either "Boost" or "The Boost Libraries".